man/sd_bus_default.xml

```diff
@@ -276,13 +276,20 @@
       <para>Returned errors may indicate the following problems:</para>
 
       <variablelist>
-
         <varlistentry>
           <term><constant>-EINVAL</constant></term>
 
           <listitem><para>The specified parameters are invalid.</para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><constant>-ENOPROTOOPT</constant></term>
```

Comment:
hmm, ENOPROTOPT is one of those errors the socket layer actually might return IRL and since we wrap the socket layer it might hit us and thus would create confusion.

Maybe pick EMEDIUMTYPE? It's not used in sd-bus yet.

Reply:
The message for EMEDIUMTYPE "Wrong medium type", which would be quite confusing for users. I don't think ENOPROTOOPT is likely to be returned. Afaics, it's only used for "invalid socket option" which is unlikely in this context, so I don't think there's much chance of confusion.

Reply:
we invoke a number of setsockopt() calls on the socket the bus connections wrap...